src/views/notifications/Components/Header.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="intro-y pr-1">
+    <div class="box p-4">
+      <div class="mb-4">
+        <h2 class="text-lg font-medium mr-auto">Latest Notifications</h2>
+        <p>Here you can view the last sent notifications</p>
+      </div>
+      <div class="chat__tabs nav nav-tabs justify-center" role="tablist">
+        <a
+          id="latest-tab"
+          :class="this.$route.name === 'moderation.notifications.latest' ? 'active bg-theme-1 text-white flex-1 py-2 rounded-md text-center self-center' : 'flex-1 py-2 border-2 rounded-md text-center self-center'"
```

Comment:
Duplicated classes on both sides of inline elseif. Rather use classes array with single elseif's.

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/4b166762d301324e361e2309c03e8e2b59b6049a